Resources/Prototypes/CharacterItemGroups/Jobs/Medical/uncategorized.yml

```
@@ -110,11 +110,14 @@
 
-#- type: characterItemGroup
-#  id: LoadoutMedicalShoes
-#  maxItems: 1
-#  items:
-#
+# WD EDIT START - uncommented
+- type: characterItemGroup
+  id: LoadoutMedicalShoes
+  maxItems: 1
+  items:
+  - type: loadout
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)